### ISSUE TITLE
Site Request

### ISSUE THREAD

**Issue by ShoaibKhanali** [SaurabhKaperwan/CSX#104]
Source name

Pahe

https://pahe.ink

_Good site with good quality please can you add it??

**Comment by yusufxz** [comment]
Can you add Turkish language movie and TV series sites similar to the Netflixmirrror site?

**Comment by SaurabhKaperwan** [comment]
> Can you add Turkish language movie and TV series sites similar to the Netflixmirrror site?

I don't make Turkish language extensions.

**Comment by yusufxz** [comment]
No, I didn't say Turkish language add-on, actually I wanted to say this: Can you add a multilingual movie and TV series site like the NetflixMirror site?

**Comment by SaurabhKaperwan** [comment]
Then tell me some websites like nfmirror. If it is possible for me to add it I will add it.

**Comment by yusufxz** [comment]
I couldn't find it, I'll let you know if I find it, but it would be great if you add it when you find it.

**Comment by rockhero1234** [comment]
Extracting link is bit complicated in pahe.ink @SaurabhKaperwan what do you think

**Comment by ryan4021** [comment]
@yusufxz 
Use this repository, it's Turkish
https://raw.githubusercontent.com/keyiflerolsun/Kekik-cloudstream/master/repo.json
